compte/service.py

```diff
@@ -59,3 +60,19 @@ def validate_from_token(activation_token, today=datetime.now(timezone.utc)):
     email_token.delete()
 
     return user, None
+
+
+def remove_personal_informations(user, uuid_gen):
+    user.email = ""
+    user.username = "<user-deleted>"
```

Comment:
If we absolutely want to stick with English here, we should update the `format_username` templatetag to handle this value and expose a French version of it.